lib/redix/set.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Redix.Set do
+  if Version.compare(System.version, "1.1.0") == :lt do
```

Comment:
Elixir 1.1 supports Erlang 17, where maps are slow, so we can't only check the Elixir version here. Given the amount of work this requires, I am leaning even more towards waiting the release of Elixir 1.4 and then bumping the requirement to `~> 1.2`, so we don't have problems with Connection and we can get rid of `HashSet` altogether. Wdyt?